### ISSUE TITLE
Feature request: cf7 push APP-NAME --strategy rolling

### ISSUE THREAD

**Issue by TheKoguryo** [nulldriver/cf-cli-resource#73]
Rolling App Deployments is GA in PAS 2.7. But to use this feature, we must to use cf CLI v7. cf CLI v7 is still beta version.

But I thought that this feature is very important. And so I request to add a command for this feature with cf cli v7. 

Thanks in advance.

### Rolling App Deployments is GA
https://docs.pivotal.io/platform/2-7/release-notes/highlights.html#rolling-app

### Commands: Rolling App Deployments
https://docs.pivotal.io/platform/application-service/2-7/devguide/deploy-apps/rolling-deploy.html#deploy

**Comment by patrickcrocker** [comment]
I've done some initial testing with the `cf7` cli against the latest version of Tanzu Application Service v2.8.5 (formerly Pivotal Application Service (formerly Pivotal Cloud Foundry)) but, unfortunately, the packaged version of the cloud controller (v3.77.0) doesn't include all of the v3 apis needed.

For instance, the `create-org` function fails with the following:
```
  [INFO] cf7 version 7.0.0-beta.30+6203bbe5d.2020-02-21
  [INFO] Executing create-org: cfclitest Org mV1dEP9aNR
  Creating org cfclitest Org mV1dEP9aNR as admin...
  OK
  
  Assigning role OrgManager to user admin in org cfclitest Org mV1dEP9aNR as admin...
  API endpoint not found at 'https://api.run.nulldriver.com/v3/roles'
  FAILED
```

**Comment by patrickcrocker** [comment]
FYI, I've added a cf7 group to the pipeline where you can track the current state of "do tests pass using the `cf7` cli": https://ci.nulldriver.com/teams/resources/pipelines/cf-cli-resource?group=cf7

**Comment by patrickcrocker** [comment]
**Update:** In cf-cli-resource [v2.19.0](https://github.com/nulldriver/cf-cli-resource/releases/tag/v2.19.0) we now experimentally support swapping out the v6 cli with v7.  Please note that at this time we have not incorporated any of the new cf cli v7 commands or functionality, rather we are simple trying to see if v7 of the cli will behave correctly.

Right now, the majority of the tests for v7 are failing due to [this issue](https://github.com/cloudfoundry/cli/issues/1966).  Once we have resolution there, we can continue to test v7 as a drop-in replacement and then look to including v7 functionality into `cf-cli-resource`.

**Comment by patrickcrocker** [comment]
Implemented in [v2.20.0](https://github.com/nulldriver/cf-cli-resource/releases/tag/v2.20.0)!

Enable by specifying `cf_cli_version: 7` in your resource `source` params and then set `strategy: rolling` in your `push` params.
